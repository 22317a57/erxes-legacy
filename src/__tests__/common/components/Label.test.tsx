import { mount, shallow } from 'enzyme';
import React from 'react';

import Label from '../../../modules/common/components/Label';

describe('Label component', () => {
  const defaultProps = {
    children: '',
    className: 'default',
    shake: true,
    ignoreTrans: true,
    lblStyle: 'default'
  };

  test('renders Label successfully', () => {
    shallow(<Label {...defaultProps} />);
  });

  test('renders successfully with default value', () => {
    const wrapper = mount(<Label {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });
});
