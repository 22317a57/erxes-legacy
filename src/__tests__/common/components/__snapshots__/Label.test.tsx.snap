// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component snapshot matches 1`] = `
<span
  className="default sc-iwsKbI jCmhMm"
>
  
</span>
`;
